query-engine/query-engine/src/server/mod.rs

```diff
@@ -67,8 +64,7 @@ pub async fn listen(opts: PrismaOpt) -> PrismaResult<()> {
     app.at("/server_info").get(server_info_handler);
     app.at("/status").get(|_| async move { Ok(json!({"status": "ok"})) });
 
-    info!("Started http server on {}:{}", addr.ip(), addr.port());
```

Comment:
Was the removal of this line conscious? Might be that the javascript relies on this.

Reply:
Yeah it was conscious since `tide` includes log functionality, but I think you're right: https://github.com/prisma/prisma/blob/afc33d2162e410836ab8eedf1205f0ed0c099056/src/packages/engine-core/src/NodeEngine.ts#L554

Good catch, will restore and add a comment this is critical to the functioning of the application.

Reply:
Restored the comment that's being matched on and added a PR to Tide to ensure the format will not change. We can PR the client at some point to switch over to Tide's format (after it's been release) and remove this statement.